core/domain/email_manager.py

```diff
@@ -1159,6 +1242,113 @@ def send_mail_to_notify_users_to_review(user_id, category):
             email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def send_mail_to_notify_contributor_dashboard_reviewers(
+        reviewer_ids, reviewers_suggestion_email_infos):
+    """Sends an email to each Contributor Dashboard reviewer notifying them of
+    the suggestions that have been waiting the longest for reivew, that the
+    reviewer has permission to review.
+
+    Args:
+        reviewer_ids: list(str). A list of the Contributor Dashboard reviewer
+            user ids to notify.
+        reviewers_suggestion_email_infos:
+            list(list(ReviewableSuggestionEmailInfo)). A list of suggestion
+            email content info objects for each reviewer. These suggestion
+            email content info objects contain the key imformation about the
+            suggestions we're notifying reviewers about and will be used to
+            compose the email body for each reviewer.
+    """
+    email_subject = NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO[
+        'email_subject']
+    email_body_template = NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO[
+        'email_body_template']
+
+    if not feconf.CAN_SEND_EMAILS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    if not config_domain.NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_IS_ENABLED:
+        log_new_error(
+            'Notifying Contributor Dashboard reviewers must be enabled on the '
+            'config page in order to send reviewers the emails.'
+        )
+        return
+
+    if not reviewer_ids:
+        log_new_error('No Contributor Dashboard reviewers to notify.')
+        return
+
+    reviewer_user_settings = user_services.get_users_settings(reviewer_ids)
+    reviewer_usernames = [
+        reviewer_user_setting.username if reviewer_user_setting is not None else
+        None for reviewer_user_setting in reviewer_user_settings
+    ]
+    reviewer_emails = [
+        reviewer_user_setting.email if reviewer_user_setting is not None else
+        None for reviewer_user_setting in reviewer_user_settings
+    ]
```

Comment:
```suggestion
    reviewer_usernames, reviewer_emails = list(zip(*[
        (reviewer_user_setting.username, reviewer_user_setting.email)
        if reviewer_user_setting is not None else None
        for reviewer_user_setting in reviewer_user_settings
    ]))
```

Alternative that makes use of zipping & destructure assignment to require only 1 list comprehension instead of two. This makes order & index consistency slightly clearer, though it should be close to equivalent to what you have now.

See: https://www.geeksforgeeks.org/python-unzip-a-list-of-tuples/.

Reply:
I didn't know that you could unzip like that - cool, thanks :)